aiohttp/client.py

```diff
@@ -611,34 +613,40 @@ async def _request(
                             resp.release()
 
                         try:
-                            parsed_url = URL(
+                            parsed_redirect_url = URL(
                                 r_url, encoded=not self._requote_redirect_url
                             )
-
                         except ValueError as e:
-                            raise InvalidURL(r_url) from e
+                            raise InvalidRedirectURL(r_url) from e
```

Comment:
Similarly, assign a description here:
```suggestion
                            raise InvalidRedirectURL(
                                r_url,
                                "Server attempted redirecting to a location that does not look like a URL",
                            ) from e
```

Reply:
added